distributor/src/distributor/save.bal

```diff
@@ -300,8 +300,6 @@ function addToPresidentialCumulative(map<json> jm) returns PresidentialCumulativ
         accum.summary.valid += <int>jm.summary.valid;
         accum.summary.rejected += <int>jm.summary.rejected;
         accum.summary.polled += <int>jm.summary.polled;
-        // don't add up electors from postal PDs as those are already in the district elsewhere
-        string pdCode = <string>jm.pd_code; // check 
```

Comment:
Do we need to remove this?

Reply:
Yeah it's nor used anymore. The logic that used this variable has been removed, but seems like the variable wasn't.